login-workflow/docs/theme.md

```diff
@@ -0,0 +1,71 @@
+# Using the Theme in your Project
+
+This guide will help you integrate and use the @brightlayer-ui/react-theme in your project.
+
+## Setup
+
+**Install Dependencies:**
+
+Install with npm
+
+```shell
+npm install --save @brightlayer-ui/react-themes
+```
+
+or yarn
+
+```shell
+yarn add @brightlayer-ui/react-themes
+```
+
+# Usage
+
+To use these themes in your application, simply wrap the app in a `ThemeProvider` and pass in the theme.:
+
+```tsx
+import { ThemeProvider, StyledEngineProvider } from '@mui/material/styles';
+import { App } from './App';
+import CssBaseline from '@mui/material/CssBaseline';
+import { blueThemes } from '@brightlayer-ui/react-themes';
+import '@brightlayer-ui/react-themes/open-sans';
+import { createRoot } from 'react-dom/client';
+
+const container = document.getElementById('root');
+const root = createRoot(container || document.createDocumentFragment());
+
+root.render(
+    <React.StrictMode>
+        <StyledEngineProvider injectFirst>
+            <ThemeProvider theme={blueThemes}>
```

Comment:
can you also add how to override direction to RTL

Reply:
I can add it, but in this workflow we have not added any configuration for RTL. Since in the example and library we are not changing direction, do we need to added override for direction toggle?

Reply:
Added RTL